src/stage1/recurse_fwd.jl

```diff
@@ -27,6 +27,10 @@ struct ∂☆new{N}; end
 function ∂☆nomethd(@nospecialize(args))
     throw(MethodError(primal(args[1]), map(primal, Base.tail(args))))
 end
+function ∂☆builtin((f_bundle, args...))
+    f = primal(f_bundle)
+    throw(DomainError(f, "No `ChainRulesCore.frule` found for the built-in function `$f`"))
```

Comment:
Let's include `args` since it depends on method signatures of `frule`s.
Some bulitins have multiple dispatch signatures.
```suggestion
    f = primal(f_bundle)
    argtypes = Any[Core.Typeof(primal(arg)) for arg in args]
    tt = Base.signature_type(f, argtypes)
    sig = Base.sprint(Base.show_tuple_as_call, Symbol(""), tt)
    throw(DomainError(f, "No `ChainRulesCore.frule` found for the built-in function `$sig`"))
```